packages/ra-ui-materialui/src/layout/ViewTitle.js

```diff
@@ -10,11 +10,14 @@ import AppBarMobile from './AppBarMobile';
 const ViewTitle = ({ className, title, ...rest }) => (
     <Responsive
         xsmall={
-            <AppBarMobile
-                className={classnames('title', className)}
-                title={title}
-                {...rest}
-            />
+            <Fragment>
+                <AppBarMobile
+                    className={classnames('title', className)}
+                    title={title}
+                    {...rest}
+                />
+                <span> </span>
```

Comment:
why the span ?

Reply:
To enable the `justify-content: space-between` to work even though the first element of the list (the title) is removed. Therefore I add another element to be the first, otherwise the Actions become the first element and are aligned left.